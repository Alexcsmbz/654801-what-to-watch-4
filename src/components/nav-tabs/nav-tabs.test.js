import NavTabs from './nav-tabs.jsx';
import NavTab from 'components/nav-tab/nav-tab.jsx';

const tab = {
  name: `Overview`,
  isActive: true,
  idx: 0,
};

describe(`NavTabs snapshot test`, () => {
  it(`NavTabs should render nav tabs`, () => {
    const tree = renderer.create(
        <NavTabs>
          <NavTab
            onClick={() => {}}
            tab={tab}
          />
          <NavTab
            onClick={() => {}}
            tab={tab}
          />
          <NavTab
            onClick={() => {}}
            tab={tab}
          />
        </NavTabs>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
