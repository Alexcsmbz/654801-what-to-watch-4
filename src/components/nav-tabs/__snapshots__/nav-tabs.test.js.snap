// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavTabs snapshot test NavTabs should render nav tabs 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item undefined"
      onClick={[Function]}
    >
      <div
        className="movie-nav__link"
      >
        Overview
      </div>
    </li>
    <li
      className="movie-nav__item undefined"
      onClick={[Function]}
    >
      <div
        className="movie-nav__link"
      >
        Overview
      </div>
    </li>
    <li
      className="movie-nav__item undefined"
      onClick={[Function]}
    >
      <div
        className="movie-nav__link"
      >
        Overview
      </div>
    </li>
  </ul>
</nav>
`;
